src/features/lightspeed/handleApiError.ts

```diff
@@ -20,38 +20,44 @@ export function retrieveError(err: AxiosError): string {
         return "Bad Request response. Please try again.";
       }
     } else if (err?.response?.status === 403) {
-      const responseErrorData = <AxiosError<{ message?: string }>>(
-        err?.response?.data
-      );
       if (
-        responseErrorData &&
-        responseErrorData.hasOwnProperty("message") &&
-        responseErrorData.message?.includes("WCA Model ID is invalid")
+        (err?.response?.headers["server"] || "").toLowerCase() === "cloudfront"
       ) {
-        return `Model ID is invalid. Please contact your administrator.`;
-      } else if (responseErrorData) {
-        switch (
-          responseErrorData.hasOwnProperty("code") &&
-          responseErrorData.code
+        return `CloudFront rejected the request. Please contact your administrator.`;
```

Comment:
Are you thinking contact your admin so they can contact support? Ultimately the admin needs to reach out to Red Hat support.

Reply:
I used the same message as we use elsewhere for consistency (e.g. Cloudflare exceptions). Happy to change (them all?) if that's the preference.

Reply:
The "Please contact your administrator." makes sense in the case of the invalid model id, since the admin presumably has access to the correct value and/or handed them the wrong value. Here though the admin can't do anything except point them to support. How about something like "Something in your editor content has caused your inline suggestion request to be blocked. Please open a ticket with Red Hat support and include the content of your editor up to the line and column where you requested a suggestion."

Reply:
Changed to match your proposal... I'm not one to procrastinate on the wording; IMO it's very subjective.

Reply:
Agree. I'm mostly just glad to recognize this as a CloudFront issue now. Thanks Michael!